README.md

```diff
@@ -209,6 +209,18 @@ e.g.
 $ TFENV_TERRAFORM_VERSION=latest:^0.11. terraform --version
 ```
 
+##### `TFENV_NETRC_PATH`
+
+String (Default: "")
+
+If not empty string, this variable specifies the credentials file used to access the remote location (useful if used in conjunction with TFENV_REMOTE).
+
+e.g.
+
+```console
+$ TFENV_NETRC_PATH="$PWD/netrc"
```

Comment:
surely `.netrc?`

Would we not use a .netrc if it were present by default?

Reply:
My reasoning was if you wanted to create one specific for the remote you are using; it would also allow you to do something like this: https://stackoverflow.com/a/33818945 which would ensure that the secret you are using is isolated from the rest of your environment.